pmm-app/.eslintrc

```diff
@@ -52,7 +52,7 @@
       ],
     // no-unused-vars doesn't work with typescript
     "no-unused-vars": "off",
-    "@typescript-eslint/no-unused-vars": ["error"],
+    "@typescript-eslint/no-unused-vars": ["off"],
```

Comment:
This was replaced with new configs added in `tsconfig.json` so we are still checking for this.